### DISCUSSION TITLE
Playbook Rule will not go away

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#7980] [Unsupported Versions] [answered]
SO2.3.110

Created playbook rule, enabled rule for testing, deactivated rule and deleted it.

The rule still generates alerts even after being deleted. How can I manually delete from playbook or check why this rule still alerts after being deleted. Any suggestions?

Cole

**Comment by dougburks** [comment, marked as answer]
Have you checked `/opt/so/rules/elastalert/playbook/` to see if the rule is still there?

**Reply by ColeVan** [reply]
Thanks @dougburks found the rule still hanging around in /opt/so/rules/elastalert/playbook/.

I removed it and restarted Playbook


Cole
